src/NServiceBus.AcceptanceTests/ScenarioDescriptors/AllTransports.cs

```diff
@@ -14,24 +14,7 @@ protected AllTransports()
             AddRange(ActiveTransports);
         }
 
-        static IEnumerable<RunDescriptor> ActiveTransports
-        {
-            get
-            {
-                if (activeTransports == null)
-                {
-                    //temporary fix until we can get rid of the "AllTransports" all together
-                    activeTransports = new List<RunDescriptor>
-                    {
-                        Transports.Default
-                    };
-                }
-
-                return activeTransports;
-            }
-        }
-
-        static ICollection<RunDescriptor> activeTransports;
+        static IEnumerable<RunDescriptor> ActiveTransports => new List<RunDescriptor> { Transports.Default };
```

Comment:
Yes it is. Because the `RunDescriptor` is now used to store settings per run if we'd cache the list you'd see the entries in the run descriptors growing per app domain and test execution. I fell into this trap while refactoring :dancers: